react/features/filmstrip/subscriber.web.js

```diff
@@ -26,7 +26,12 @@ import './subscriber.any';
  * Listens for changes in the number of participants to calculate the dimensions of the tile view grid and the tiles.
  */
 StateListenerRegistry.register(
-    /* selector */ getParticipantCountWithFake,
+    /* selector */ state => {
+        return {
```

Comment:
Selectors compare objects with simple euality checks, you need to pass the extra parameter to register to is does a deep equality check.